pom.xml

```diff
@@ -6,6 +6,7 @@
         <groupId>org.jboss.tools</groupId>
         <artifactId>parent</artifactId>
         <version>4.4.1.Alpha1-SNAPSHOT</version>
+        <relativePath/>
```

Comment:
I guess that I still don't understand the issue. Maven will still try to resolve it from the local repository and remote repositories if there isn't a pom in ../pom.xml.